apple/Sources/Sargon/Extensions/Methods/Prelude/NetworkDefinition+Wrap+Functions.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Cyon on 2024-04-22.
+//
+
+import Foundation
+import SargonUniFFI
+
+extension NetworkDefinition {
+	public static func lookupBy(logicalName: String) throws -> Self {
+		try newNetworkDefinitionLookupByName(logicalName: logicalName)
```

Comment:
I was thinking about that on Android when we receive a logical name from GW. Should we use this to export a network definition?

I did this:
```
val response: GatewayStatusResponse = ...

val networkId =NetworkId.entries.find { it.string == response.ledgerState.network }
val gateway = Gateway.init(newUrl, info.id) // newGatewayWithUrlOnNetwork(...)
```

Reply:
Same thing,but yes best use Sargon